43-graphql/README.md

```diff
@@ -0,0 +1,444 @@
+
+<details>
+<summary>query</summary>
+```graphql
+query {
+  allPokemons {
+    id
+    name
+  }
+}
+```
+</details>
+
+If you want to follow along with the queries we use, head over to [the interactive GraphiQL tool](https://api.graph.cool/simple/v1/ciwjew0qz0l8d0122v7smvmxu) connecting you to the pokemon project that we setup for this article. If you copy the above query and hit the play button, you should see this result:
+
+<details>
+<summary>response</summary>
+```json
+{
+  "data": {
+    "allPokemons": [
+      {
+        "id": "ciwnmyvxn94uo0161477dicbm",
+        "name": "Pikachu"
+      },
+      {
+        "id": "ciwnmzhwn953o0161h7vwlhdw",
+        "name": "Squirtle"
+      },
+      {
+        "id": "ciwnn0kxq95oy0161ib2wu50g",
+        "name": "Bulbasaur"
+      },
+      {
+        "id": "ciwnn11i1960l0161861mxdc1",
+        "name": "Charmander"
+      }
+    ]
+  }
+}
+```
+</details>
+
+The GraphiQL tool is maintained by Facebook and highlights another benefit of GraphQL. The GraphQL schema contains so called introspection queries that make all information about the type system available. This allows for the automatically generated documentation in the top right corner of the tool or the great auto-complete feature.
+
+Now go ahead and use the auto-complete feature of GraphiQL with `Ctrl+Space` to add the `url` field to the `allPokemons` query, run it and examine the result. Woah! Changing queries is as easy as that - just include whatever fields you want and run the query. There's one more crucial property of queries. Let us run the following query together:
+
+<details>
+<summary>query</summary>
+```graphql
+query {
+  allPokemons {
+    id
+    name
+    trainer {
+      name
+    }
+  }
+}
+```
+</details>
+
+You should see this result:
+
+<details>
+<summary>response</summary>
+```json
+{
+  "data": {
+    "allPokemons": [
+      {
+        "id": "ciwnmyvxn94uo0161477dicbm",
+        "name": "Pikachu",
+        "trainer": {
+          "name": "Ash Ketchum"
+        }
+      },
+      {
+        "id": "ciwnmzhwn953o0161h7vwlhdw",
+        "name": "Squirtle",
+        "trainer": {
+          "name": "Ash Ketchum"
+        }
+      },
+      {
+        "id": "ciwnn0kxq95oy0161ib2wu50g",
+        "name": "Bulbasaur",
+        "trainer": {
+          "name": "Ash Ketchum"
+        }
+      },
+      {
+        "id": "ciwnn11i1960l0161861mxdc1",
+        "name": "Charmander",
+        "trainer": {
+          "name": "Ash Ketchum"
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+Just like with normal fields, we can even select related nodes in our query. This dramatically decreases the amount of queries needed and allows for quick prototyping and development of frontend applications. Instead of the time-draining skimming through dozens of potential endpoints, you can simply explore the possible queries interactively which has a great effect on developer experience.
```

Comment:
Minor: "This dramatically decreases the _amount_ of queries needed" should be "This dramatically decreases the _number_ of queries needed"